CMakeLists.txt

```diff
@@ -30,11 +29,78 @@ option(
     ${PROJECT_IS_TOP_LEVEL}
 )
 
-include(CTest)
+add_library(beman.exemplar)
+add_library(beman::exemplar ALIAS beman.exemplar)
+
+# gersemi: off
+set_target_properties(
+    beman.exemplar
+    PROPERTIES
+        VERIFY_INTERFACE_HEADER_SETS ON
+        EXPORT_NAME exemplar
+)
+
+
+target_sources(
+    beman.exemplar
+    PRIVATE
+        src/beman/exemplar/identity.cpp
+
+    PUBLIC
+        FILE_SET HEADERS
+        BASE_DIRS include
+        FILES
+            include/beman/exemplar/identity.hpp
+
+    PUBLIC
+        FILE_SET CXX_MODULES
+        BASE_DIRS include
+        FILES
+            include/beman/exemplar/beman.exemplar.cppm
+)
+
+include(GNUInstallDirs)
 
-add_subdirectory(src/beman/exemplar)
+install(
+    TARGETS beman.exemplar
+    EXPORT beman.exemplar-targets
+    COMPONENT beman.exemplar
+
+    FILE_SET CXX_MODULES
+        DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
```

Comment:
is this intended to install the `cppm` files to `include` tree?

Reply:
Yes.

Reply:
There has been ongoing arguments about where they should go, since they aren't files that one should #include, but there are a two packages that have the module file in include, vulkan and gcc-15 if I recall correctly, so there's some precedent. It doesn't really matter terribly as long as the metadata file for the module points to the location so the module can be compiled in the context of the consumer, with the consumers compiler, flags, etc.

Reply:
The exist outside the bounds of what is defined by the FHS. In spirit they are closest to header files, being architecture-independent interface descriptions, so they go in `include`.